src/App.js

```diff
@@ -1,13 +1,27 @@
 import React from 'react';
+import {
+  Routes,
+  Route,
+} from 'react-router-dom';
+import Header from './components/Header';
 import Calculator from './components/Calculator';
+import Home from './pages/Home';
+import Quote from './pages/Quote';
 
 // eslint-disable-next-line react/prefer-stateless-function
```

Comment:
Please ensure that linters are not disabled.  A linter is a tool that analyzes your source code to flag programming errors, bugs, stylistic errors, and suspicious constructs(source: [Wikipedia](https://en.wikipedia.org/wiki/Lint_(software))).

Reply:
This line is needed as otherwise, the linter error can not be avoided.